docs/accessibility.mdx

```diff
@@ -0,0 +1,36 @@
+---
+name: Accessibility
+route: /a11y
+---
+
+import { TitleCard } from './components/TitleCard';
+
+<TitleCard title="Accessibility (a11y)">
+    Accessible design not only helps users with disabilities; it provides better user experiences for everyone.
+</TitleCard>
+
+## Overview
+
+Accessible design not only helps users with disabilities; it provides better user experiences for everyone. Our goal is to provide a consistent user experience for all users, whether they’re navigating with a mouse/touchscreen, keyboard, or screen reader.
+
+An accessible product should:
+
+- Give all users the same quality of experience
+- Adapt to users and situations
+
+Individually accessible elements and components are part of building accessible products, but the accessibility efforts need for manual testing. When designing new UIs, rely as much as possible on Wave existing components and follow accessible patterns. If you're creating a new component or pattern (one that doesn't exist in Wave), keep in mind accessibility practices before your implementation.
+
+## Mission
```

Comment:
I also think about compliance with the WCAG AA Level, but let's think it through first because it implies a change of the development process